x-pack/docs/en/security/limitations.asciidoc

```diff
@@ -62,11 +63,12 @@ When a user's role enables document level security for a data stream or index:
 * The `has_child` and `has_parent` queries aren't supported as query in the
   role definition. The `has_child` and `has_parent` queries can be used in the
   search API with document level security enabled.
-* Any query that makes remote calls to fetch data to query by isn't supported.
-  The following queries aren't supported:
-** The `terms` query with terms lookup isn't supported.
-** The `geo_shape` query with indexed shapes isn't supported.
-** The `percolate` query isn't supported.
+* Any query that makes remote calls to fetch query data isn't supported,
+including the following queries:
+** `terms` query with terms lookup
+** `geo_shape` query with indexed shapes
+** `percolate` query
+** `date_math_expr` range query that includes `now`
```

Comment:
Date math does not make remote calls. So it should not be nested under the title of "query that makes remote calls". Also I don't think we have an official name of "date_math_expr range query". The [relevant doc page](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html#ranges-on-dates) describes it as "using the `range` query with `date` fields. So maybe we could say something like:

[Date math](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#date-math) expressions cannot contain `now` in [range query with date fields](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html#ranges-on-dates).